dspace-api/pom.xml

```diff
@@ -467,6 +467,14 @@
             <groupId>org.apache.poi</groupId>
             <artifactId>poi-scratchpad</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk15on</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcmail-jdk15on</artifactId>
+        </dependency>
```

Comment:
Hi @ilittle-cnri :  Could you clarify how/why these BouncyCastle dependencies are needed for running the Handle Server?

Removing these dependencies was not accidental.  I removed them in PR #2425 , as we were looking to clean up dependencies that had known security issues.

While we could add them back in here, I wanted to first make sure I understand why they are required by the Handle Server.  If there's no way around including them, we should add a comment here to make it clear where they are used/needed, as the DSpace code doesn't use BouncyCastle itself (at all).  So, I admit, it's slightly confusing why the Handle Server isn't including these dependencies itself, instead of depending on DSpace to include them for it.